packages/shortest/src/cli/bin.ts

```diff
@@ -205,7 +194,7 @@ const main = async () => {
     log.trace("Initializing TestRunner");
     const runner = new TestRunner(process.cwd(), config);
     await runner.initialize();
-    const success = await runner.runTests(testPattern);
+    const success = await runner.execute(config.testPattern);
```

Comment:
This name better represents what the function does (at the time of calling, is not known yet if there are any tests to be executed yet).